README.md

```diff
@@ -570,6 +570,25 @@ const observer = {
 meetingSession.audioVideo.addObserver(observer);
 ```
 
+**Use case 17.** Add an observer to know when remote video sources get changed.
+
+> Note: `getRemoteVideoSources` method is different from `getAllRemoteVideoTiles`, 
+`getRemoteVideoSources` returns all the remote video sources that are available to be viewed,
+while `getAllRemoteVideoTiles` returns the ones that are actually being seen.
+
+```js
+const observer = {
+  remoteVideoSourcesDidChange: attendees => {
+    console.log(`available remote videos changed: ${JSON.stringify(attendees)}`);
```

Comment:
I am updating this use case, please let me know if this is fine now? I was not sure on what is the correct way to tell about the `Attendee` object here since other use case do not say about type of returned data specifically like `videoTileState` etc. (Also I will update `Attendee` to finalized name later once confirmed)

**Use case 17.** Add an observer to know all the remote video sources when changed.
> Note: `remoteVideoSourcesDidChange` callback is called with array of Attendee objects.
Check [Attendee](https://aws.github.io/amazon-chime-sdk-js/classes/attendee.html) for more information.
```js
import { Attendee } from 'amazon-chime-js-sdk';

const observer = {
  remoteVideoSourcesDidChange: attendees => {
    // attendees is an array of Attendee objects with each object holding remote video source information.
    // [{attendeeId: '1', externalUserId: 'ext#1'}, {attendeeId: '2', externalUserId: 'ext#2'}, ...]
    console.log('available remote videos changed: ', attendees);
  }
};

meetingSession.audioVideo.addObserver(observer);
```
You can also call below method to know all the remote video sources:
> Note: `getRemoteVideoSources` method is different from `getAllRemoteVideoTiles`, 
`getRemoteVideoSources` returns all the remote video sources that are available to be viewed,
while `getAllRemoteVideoTiles` returns the ones that are actually being seen.
```js
const remoteVideoSources = meetingSession.audioVideo.getRemoteVideoSources();
```

Reply:
Discussed offline and updated.